### ISSUE TITLE
[BUG] Git repo's about sidebar points to non-existent GitHub pages URL

### ISSUE THREAD

**Issue by pushfoo** [Marco-Sulla/python-frozendict#85]
### tl;dr

The repo homepage points to a  GitHub pages link which does not exist: https://marco-sulla.github.io/python-frozendict/

### Replication steps

1. Go to the repo home page
2. Click the URL in the About section:
    ![image](https://github.com/Marco-Sulla/python-frozendict/assets/36696816/22524f1d-0a19-44fe-b2eb-f252bf82c725)
3. Observe 404:
    ![image](https://github.com/Marco-Sulla/python-frozendict/assets/36696816/09211919-5305-40d1-a29a-997a9e24236b)

### Potential fixes

There doesn't seem to be a readthedocs for this project and the README.md seems to be the most authoritative source. Perhaps removing the link would be best if the doc won't be restored?

**Comment by Marco-Sulla** [comment]
Confirmed.

**Comment by Marco-Sulla** [comment]
Fixed. It was simply the Github description, nothing in the README.


> There doesn't seem to be a readthedocs for this project

The project is so small I didn't feel the urgency to add a readthedocs. Feel free to open an enhancement proposal.
